src/os/ui/feature/simpleproperties.js

```diff
@@ -0,0 +1,244 @@
+goog.module('os.ui.feature.simpleproperties');
```

Comment:
This module is just a standard UI/controller directive, and it should follow the naming conventions (i.e. `os.ui.feature.SimplePropertiesUI`).